packages/backend/package.json

```diff
@@ -35,8 +36,8 @@
         "@casl/ability": "^5.4.3",
         "@cubejs-client/core": "^0.35.23",
         "@godaddy/terminus": "^4.12.0",
-        "@lightdash/common": "^0.1440.0",
-        "@lightdash/warehouses": "^0.1440.0",
+        "@lightdash/common": "workspace:*",
```

Comment:
what's this syntax ?

Reply:
if this works, this would also solve the conflicts on commercial

Reply:
yes exactly! have a read, it's so nice! https://pnpm.io/workspaces#workspace-protocol-workspace